### DISCUSSION TITLE
Easier debug while loading

### DISCUSSION THREAD

**Discussion by ghost** [IndiePython/nodezator#91] [Minor issue]
Atm. I can't get any prints to show up anywhere to debug this loading issue. Sure, I can run the code other ways but it's quite inconvenient.

![image](https://github.com/IndiePython/nodezator/assets/29675001/a859cb13-f04d-436c-a773-12c4e394a56b)

**Comment by KennedyRichard** [comment]
The patch release that fixed issue #88 also improved error reporting. The full error report can now be seen in the user log, by pressing **Shift+Ctrl+j** when you are on the canvas/graph, just like you would do to bring the console in a browser. You can also access it on the menubar option **Help > Show user log**.

However, despite the issue being solved, the error report itself is still not detailed enough (for instance, we still only show the error message but not the traceback), so this is indeed a valid issue (though a minor one, since it doesn't crash the app nor obstruct usage).

Because of that, I'll keep this discussion alive.

For now, I'll prioritize other features/fixes of higher urgency/importance and only visit this minor issue at a later date.

I myself also have things I want to improve in the future for error reporting in Nodezator and I'm keeping track of them.

Thank you for the report.

Peace.
